library/ssl_tls13_server.c

```diff
@@ -739,10 +739,21 @@ static int ssl_tls13_process_client_hello( mbedtls_ssl_context *ssl )
     parse_client_hello_ret = ret;
     MBEDTLS_SSL_PROC_CHK( ssl_tls13_postprocess_client_hello( ssl ) );
 
-    if( parse_client_hello_ret == SSL_CLIENT_HELLO_OK )
-        mbedtls_ssl_handshake_set_state( ssl, MBEDTLS_SSL_SERVER_HELLO );
-    else
-        mbedtls_ssl_handshake_set_state( ssl, MBEDTLS_SSL_HELLO_RETRY_REQUEST );
+    switch( parse_client_hello_ret )
```

Comment:
I don't think we need that: see https://github.com/Mbed-TLS/mbedtls/pull/5707/#discussion_r867669893.